apps/src/aichat/views/aichatView.module.scss

```diff
@@ -13,8 +13,6 @@
   overflow: hidden;
   user-select: none;
   display: flex;
-  flex-direction: row;
-  gap: 2px;
   margin: 2px;
 
   .instructionsArea {
```

Comment:
Looking at mockup, it seems like instructions and customization panels are about the same width. https://codedotorg.slack.com/archives/C06FELVTV0Q/p1711150032784399
What do you think about assigning the width of both areas to something like `30%`?

Reply:
Also - this is outside of scope of this PR, but observing that the button is currently not in view in the 'Publish' tab. I tried adding `overflow-y` to `cutomizationArea` but unsure why it didn't work.

Reply:
ugh, getting headers and footers to stick on smaller screens will be the death of me! yeah I'd like to get this merged to avoid more merge conflicts, can follow up with changes dealing with buttons disappearing and width of the different panels, if that's ok with you.

Reply:
That's totally fine with me - thanks!